java/src/jmri/jmrix/cmri/serial/sim/configurexml/ConnectionConfigXml.java

```diff
@@ -33,6 +33,11 @@ protected void getInstance() {
     }
 
     @Override
+    protected void getInstance(Object object) {
```

Comment:
because that is what the method we are Overriding takes as a parameter.  See jmri.jmrix.configurexml.AbstractSerialConnectionConfigXml